src/pages/Login/Login.js

```diff
@@ -30,13 +35,18 @@ export default function () {
 			localStorage.setItem('authToken', bearerToken);
 			axiosClient.defaults.headers.common['Authorization'] = bearerToken;
 
+			setToken(bearerToken);
 			setIsLoading(false);
```

Comment:
```suggestion
			setIsLoading(true);
```
Khúc ni mình set thành true mới đúng phải hok cu Hiếu he

Reply:
false đó, response trả về thì ko loading mới đúng chứ

Reply:
à, rứa anh phải đọc lại doc của cái thư viện ni roi vại